### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by richlander**
This page got missed with #9363.

### CODE REVIEW

File: release-notes/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # .NET Release Notes
 
-The following [.NET releases](../releases.md) are currently supported:
+The following table lists [releases](../releases.md) under active development or support:
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 9](9.0/README.md) | November 12, 2024 | [STS][policies] | [9.0.0-preview.5][9.0.0-preview.5] |  |
+| [.NET 9](9.0/README.md) | November 12, 2024 | [STS][policies] | [9.0.0-preview.5][9.0.0-preview.5] | May 11, 2026 |
```
Comment (mairaw) [inline release-notes/README.md:7]: wouldn't it be May 12? we usually end our support on patch tuesday.

Comment (richlander) [reply release-notes/README.md:7]: I tried do this correctly and still got it wrong! Yes, May 12 is correct.

Comment (richlander) [reply release-notes/README.md]: Thanks for the catch! I think I got them all.

**Review by mairaw — APPROVED** [review]
I like the consistency now 😄

**Pull request merged**